core/tests/webdriverio_desktop/contributorAdminDashboard.js

```diff
@@ -0,0 +1,351 @@
+    explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+    explorationEditorMainTab = explorationEditorPage.getMainTab();
+    storyEditorPage = new StoryEditorPage.StoryEditorPage();
+    topicEditorPage = new TopicEditorPage.TopicEditorPage();
+    creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage();
+
+    await users.createUser(TRANSLATION_COORDINATOR_EMAIL, 'translation');
+    await users.createUser(QUESTION_COORDINATOR_EMAIL, 'question');
+    await users.createUser(QUESTION_ADMIN_EMAIL, QUESTION_ADMIN_USERNAME);
+    await users.createUser(USER_EMAILS[0], 'user0');
+    await users.createUser(USER_EMAILS[1], 'user1');
+
+    await users.createAndLoginCurriculumAdminUser(ADMIN_EMAIL, 'management');
+    await adminPage.get();
+    await adminPage.addRole('management', 'release coordinator');
+    await adminPage.addRole('question', 'question coordinator');
+    await adminPage.addRole('translation', 'translation coordinator');
+    await adminPage.addRole('question', 'translation admin');
+    await users.logout();
+
+    await users.login(QUESTION_COORDINATOR_EMAIL);
+    await contributorDashboardAdminPage.get();
+    await contributorDashboardAdminPage.assignQuestionContributor('user0');
+    await contributorDashboardAdminPage.assignQuestionContributor('user1');
+    await contributorDashboardAdminPage.assignQuestionReviewer('user1');
+    await contributorDashboardAdminPage.assignTranslationReviewer(
+      'user1', 'shqip (Albanian)');
+    await users.logout();
+
+    // Populating Dashboard.
+    await users.createUser('dummy@example.com', 'dummy');
+
+    await users.login(ADMIN_EMAIL);
+    await topicsAndSkillsDashboardPage.get();
+    await topicsAndSkillsDashboardPage.createTopic(
+      TOPIC_NAMES[0], 'community-topic-one', 'Topic description 1', false);
+    const URL = await browser.getUrl();
+    // Example URL: http://localhost:8181/topic_editor/jT9z3iLnFjsQ#/
+    const TOPIC_ID_URL_PART = URL.split('/')[4];
+    // We have to remove the ending "#".
+    const TOPIC_ID = TOPIC_ID_URL_PART.substring(
+      0, TOPIC_ID_URL_PART.length - 1);
+    await workflow.createSkillAndAssignTopic(
+      SKILL_DESCRIPTIONS[0], REVIEW_MATERIALS[0], TOPIC_NAMES[0]);
+    await topicsAndSkillsDashboardPage.get();
+    await topicsAndSkillsDashboardPage.createSkillWithDescriptionAndExplanation(
+      SKILL_DESCRIPTIONS[1], REVIEW_MATERIALS[1]);
+
+    await adminPage.get();
+    await adminPage.addRole(QUESTION_ADMIN_USERNAME, 'question admin');
+    // Add topic to classroom to make it available for question contributions.
+    await adminPage.editConfigProperty(
+      'The details for each classroom page.',
+      'List',
+      async function(elem) {
+        elem = await elem.editItem(0, 'Dictionary');
+        elem = await elem.editEntry(4, 'List');
+        elem = await elem.addItem('Unicode');
+        // eslint-disable-next-line oppia/e2e-action
```

Comment:
Here lint check prevents us from directly using `elem.setValue()` it suggests to use `action.setValue()` but if I change it, it doens't seem to set the value correctly and tests fail.

Reply:
`action.setValue` internally calls `elem.setValue` so it should work, can you please check what's going on? In general, we avoid having flags and use the actions to avoid unexpected flakiness in CI.

Reply:
I think the reason is elem is a param for the async function and on passing it to `action.setValue` it is loosing context.

Reply:
I'm finding it difficult to understand, it would be great if you could explain it in detail. 

Regarding the `function` in context is `async`, that's because we are using `await` inside, right? The entire `beforeAll` section is in `async` (line 74).

Reply:
Actually the passing the value of clickable element as `false` while calling action.setValue does the work.

Reply:
Yeahs, so basically we don't want to click the element before setting the value! Thanks for looking into it and fixing it! :)

(We try to avoid using pragmas/flags, in case we do, we write proper documentation/code-comment on why it's needed. Also, developers copy code from one place to another and the pragma/flags gets copied so that was the reason I was trying to get rid of it. I hope you can understand! :) )

Reply:
Yeah that's understandable and it's always better to follow good coding practices. I am learning new things here. Thanks.